src/runtime.py

```diff
@@ -93,7 +94,8 @@ def goto(target_position: Vector):
         'calculate_distance_to': lambda id: calculate_distance_to(scene, id, object_id),
         'random_direction': lambda: random_direction(scene, object_id),
         'print': lambda text: send(Print(str(text), scene.get_object_by_id(object_id).owner)),
-        'take': lambda vector: send(Take(object_id, vector))
+        'take': lambda vector: send(Take(object_id, vector)),
+        'drop': lambda target_position: send(Drop(object_id, target_position)),
```

Comment:
No `,` needed on the end of this entry, because it is the last one

Reply:
It useful when we compare the difference between changes.
https://stackoverflow.com/questions/11597901/why-are-trailing-commas-allowed-in-a-list